pkg/commands/backend/create.go

```diff
@@ -1,6 +1,7 @@
 package backend
 
 import (
+	_ "embed"
```

Comment:
This is an example of adding a NOTE to the CLI help output for this particular command. Three things have to happen...

1. Import the embed package
2. Create a variable to hold the embedded file as a string.
3. Define a `Notes()` method that will override the default no-op version in the base object.

If no `Notes()` method defined, then nothing is added to the CLI help output.